test/pgtest/copy-from-csv.pt

```diff
@@ -0,0 +1,139 @@
+# Note that this file will not work on PG due to the lack of "strict" parsing
```

Comment:
Then this should go in the pgtest-mz directory.

Reply:
Derp! tysm for catching that